docker/Dockerfile

```diff
@@ -1,12 +1,14 @@
 FROM ubuntu:16.04
 MAINTAINER koalaman <koalaman@gmail.com>
 
-RUN apt-get update -qqy
+RUN apt-get update -qy
 
-RUN apt-get -qqy install haskell-platform
+RUN apt-get -qy install git cabal-install
+
+RUN cd $HOME && git clone https://github.com/koalaman/shellcheck.git
```

Comment:
Doesn't this clone a new copy from github rather than using the source code you currently have and may have modified?